README.md

```diff
@@ -27,20 +27,28 @@ More details on our creation process can be found in the provided reference and
 
 [^1]: D. Ziemke, I. Kaddoura, K. Nagel; [The MATSim Open Berlin Scenario: A multimodal agent-based transport simulation scenario based on synthetic demand modeling and Open Data](https://doi.org/10.1016/j.procs.2019.04.120); Procedia Computer Science, Volume 151, 2019, 870-877
 
+As an additional effort, the scenario was extended with bus traffic based on real GTFS feeds. 97\% of all bus lines in Berlin 
```

Comment:
This is only half true.
This number counts a line as being present even if only one of the two directions could be converted. I think, going with the 89% of 344 of 386 routes is more fair